gcc/cp/cp-tree.h

```diff
@@ -9001,6 +9013,30 @@ set_contract_const (tree t, bool constify)
   TREE_LANG_FLAG_4 (CONTRACT_CHECK (t)) = constify;
 }
 
+/* Test if EXP is a contract const wrapper node.  */
+
+inline bool
+contract_const_wrapper_p (const_tree exp)
+{
+  /* A wrapper node has code VIEW_CONVERT_EXPR, and the flag
+     EXPR_LOCATION_WRAPPER_P is set.  */
+  if (TREE_CODE (exp) == VIEW_CONVERT_EXPR
+      && EXPR_CONTRACT_CONST_WRAPPER_P (exp))
+    return true;
+  return false;
+}
+
+/* Implementation of STRIP_ANY_CONTRACT_CONST_WRAPPER.  */
+
+inline tree
+tree_strip_any_contract_const_wrapper (tree exp)
```

Comment:
strip_contract_constify_expr()
would match my suggestion for the query. That's what I would suggest to be the general naming scheme, name it and things related to it "contract_constify_expr".

Reply:
done